tests/tracking/_tracking_service/test_utils.py

```diff
@@ -358,3 +361,14 @@ def test_store_object_can_be_serialized_by_pickle(tmpdir):
     pickle.dump(_get_store("https://example.com"), io.BytesIO())
     # pickle.dump(_get_store(f"sqlite:///{tmpdir.strpath}/mlflow.db"), io.BytesIO())
     # This throws `AttributeError: Can't pickle local object 'create_engine.<locals>.connect'`
+
+
+@pytest.mark.parametrize("absolute", [True, False], ids=["absolute", "relative"])
+def test_set_tracking_uri_with_path(tmp_path, monkeypatch, absolute):
+    monkeypatch.chdir(tmp_path)
+    path = Path("foo/bar")
+    if absolute:
+        path = tmp_path / path
```

Comment:
```suggestion
    path = Path("foo/bar")
    if absolute:
        path = path.absolute()
```

Can we use `absolute()` here?

Reply:
I prefer creating a temporary directory and working in that rather than mess with the current path which is different for each user.

Reply:
Makes sense!